arcgis-ios-sdk-samples/Augmented reality/Collect data in AR/CollectDataAR.storyboard

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="14490.70" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="eeF-1C-KCv">
+    <device id="retina6_1" orientation="portrait">
+        <adaptation id="fullscreen"/>
+    </device>
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="14490.49"/>
+        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Collect DataAR-->
+        <scene sceneID="ihT-Qv-am2">
+            <objects>
+                <viewController id="eeF-1C-KCv" customClass="CollectDataAR" customModule="ArcGISToolkitExamples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="XZF-OQ-lPz">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="H1P-pM-V3w" customClass="ArcGISARView" customModule="ArcGISToolkit">
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="dpR-4S-84i">
+                                <rect key="frame" x="0.0" y="813" width="414" height="49"/>
+                                <items>
+                                    <barButtonItem title="Calibrate" id="3DT-NA-fEY">
+                                        <connections>
+                                            <action selector="showCalibrationPopup:" destination="eeF-1C-KCv" id="9dD-t4-50W"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="bT6-cL-gQC"/>
+                                    <barButtonItem style="plain" id="gCU-57-zVn">
+                                        <segmentedControl key="customView" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="top" segmentControlStyle="bar" selectedSegmentIndex="0" id="SCJ-de-Koy">
```

Comment:
I don't see that in Xcode 10.2.  I'm not sure how it can be misplaced, as it's part of the Toolbar and placed next to Flexible Space items so it's location is determined dynamically.

Reply:
I saw this in Xcode 11.0. I'll take another look with your latest changes.

Reply:
I'm still seeing this warning after pulling your latest changes. Since it is only visible in Xcode 11, I suspect it is related to UI changes in iOS 13.

Reply:
So are you OK with it as-is?

Reply:
Since Xcode 11.0 is now the minimum requirement, I would recommend opening the storyboard in Xcode 11 and fixing the misplacement by updating the frames:

<img width="672" alt="image" src="https://user-images.githubusercontent.com/2257493/66334816-9a4b5700-e8ee-11e9-8371-dc916518b13a.png">

Reply:
Done.